common/main/ug.xml

```diff
@@ -1157,6 +1157,7 @@ Reviewed by Waris Abdukerim Janbaz <oyghan@gmail.com> of the Uyghur Computer Sci
 		<exemplarCharacters type="auxiliary">[\u200E\u200F ئ]</exemplarCharacters>
 		<exemplarCharacters type="index" draft="contributed">[{ئا} {ئه} {ئو} {ئۇ} {ئۆ} {ئۈ} {ئې} {ئى} ا ە ب پ ت ج چ خ د ر ز ژ س ش غ ف ق ك گ ڭ ل م ن ھ و ۇ ۆ ۈ ۋ ې ى ي]</exemplarCharacters>
 		<exemplarCharacters type="numbers">↑↑↑</exemplarCharacters>
+		<exemplarCharacters type="punctuation">[\- ‑ , ; : ! ? . ( ) \[ \] { } ‹ › « »]</exemplarCharacters>
```

Comment:
@conradarcturus i think `{` and `}` need to be escaped. Are they really used here?

Reply:
Great! Nice to see the error message. I'll escape it.

I included the curly braces because they were inherited from root. https://st.unicode.org/cldr-apps/v#/ug/Alphabetic_Information/6cf943e652b01478